version1/02 - Object types.md

```diff
@@ -90,863 +94,638 @@ The following notation is used in the tables below:
+| M108      | date      | arkivperiodeSluttDato             |          |             |                |  |  V   |  V   |   V   |  V   |      |        | <p>&bull; Mandatory if arkivdelstatus is set to overlapping period</p>                                                                           |
+|           | date      | overlappingsperiodeSluttdato      |          |             |                |  |  V   |  V   |   V   |  V   |      |        |                                                                                                                                                  |
+| M500      | string    | skjerming                         |          |             | skjerming      |  |  V   |  V   |   V   |      |      |        |                                                                                                                                                  |
+|           |           |                                   |          |             |                |  |      |      |       |      |      |        |                                                                                                                                                  |
+|           | ref       | refArkiv                          |    V     |             |                |  |      |  V   |   V   |      |  V   |   V    |                                                                                                                                                  |
+|           | ref       | refPrimaerKlassifikasjonssystem   |          |             |                |  |      |  V   |   V   |      |  V   |   V    | <p>Mandatory:</p><p>&bull; If null inherited from a child</p><p>&bull; If not null all childs must have the same primary KS </p>;                |
+|           | ref       | refSekundaerKlassifikasjonssystem |          |             |                |  |      |      |   V   |      |  V   |   V    |                                                                                                                                                  |
+|           | ref       | refMappe                          |          |             |                |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **AbstraktMappe**</p>                                                                                                       |
+|           | ref       | refRegistrering                   |          |             |                |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **Basisregistrering**</p>                                                                                                   |
+|           | ref       | refArkivdelForloeper              |          |             |                |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **Arkivdel**. Cross references are not allowed</p><p>&bull; Mandatory if arkivdelstatus is set to overlapping period</p> |
```

Comment:
The comment here is incorrect, it is not mandatory to have the field set when arkivdelstatus is set to overlapping period. That is only true for the `refArkivdelArvtaker` field below.